rateapp/src/css/main.css

```diff
@@ -0,0 +1,192 @@
+@tailwind base;
+@tailwind components;
+
+/* General Layout */
+
+#app {
+    height: 100vh;
+    overflow: hidden;
+}
+
+.navbar {
+    @apply flex content-center justify-between border-b border-black px-4 py-2
+}
+
+.rate, .find {
+    height: calc(100vh - 69px);
+}
+
+.search {
+    @apply bg-gray-100;
+}
+
+.results {
+    height: calc(100vh - 188px);

```

Comment:
Cool, werkt slecht in IE11 en die is nog steeds officieel supported. Dus altijd goed vooraf met de klant checken of dit een optie is. Bij dit project geen probleem, maar waarschijnlijk wel bij andere projecten van SURF.

Reply:
Thanks, ga ik meenemen. Moet eerlijk toegeven dat ik niet vaak genoeg rekening houd met IE11, maar goed om in achterhoofd te houden idd.